api/src/test/java/org/openmrs/util/OpenmrsUtilTest.java

```diff
@@ -666,9 +688,156 @@ public void shortenedStackTrace_shouldReturnNullIfStackTraceIsNull() {
 	}
 	
 	/**
-	 * @see OpenmrsUtil#nullSafeEqualsIgnoreCase(String,String)
+	 * @see OpenmrsUtil#folderContains(File, String)
+	 */
+	@Test
+	public void folderContains_shouldReturnTrueWhenFolderContainsFileWithGivenName() {
+		File folder = createFileInUserDirectory("folder");
+		folder.mkdir();
+		File file = createFileInUserDirectory("folder" + separator + "file");
+		try {
+			file.createNewFile();
+			assertTrue(OpenmrsUtil.folderContains(folder, "file"));
+			file.delete();
+
+		} 
+		catch (IOException e) {Assert.fail("Unable to create file");}
+		folder.delete();
+	}
+	
+	/**
+	 * @see OpenmrsUtil#folderContains(File, String)
 	 */
 	@Test
+	public void folderContains_shouldReturnFalseWhenFolderDoesNotContainFileWithGivenName() {
+		File folder = createFileInUserDirectory("folder");
```

Comment:
Was this(createFileInUserDirectory) method introduced for creating a file/folder or both?

Reply:
This method create file. If I want to create directors I just call mkdir() on this file. I can't modify this to do both because this would add to much complexity (one more argument, at least one if etc). So it's better to add a line of code when I want folder (I thinks I used it two times.